README.md

```diff
@@ -136,5 +136,20 @@ to
 |1        | 0.4        | 0.7         |
 
 ### Train-Test-Split
+Splitting our dataset is essential for an unbiased evaluation of prediction performance.we split randomly our dataset into 3 subset:
+- Training set : is applied to train, or fit, the model
+- Validation set :is used for unbiased model evaluation during hyperparameter tuning
+- Test set:is needed for an unbiased evaluation of the final model.
```

Comment:
Training set: is applied ...
Validation set: is used for ...
Test set: is needed ...
-> just check the whitespaces